app/src/features/surveys/view/survey-animals/form-sections/FamilyAnimalForm.tsx

```diff
@@ -41,176 +38,140 @@ const useStyles = makeStyles((theme: Theme) => ({
   return (
-    <FieldArray validateOnChange={true} name={name}>
-      {({ remove, push }: FieldArrayRenderProps) => (
-        <>
-          <FormSectionWrapper
-            title={SurveyAnimalsI18N.animalFamilyTitle}
-            addedSectionTitle={SurveyAnimalsI18N.animalFamilyTitle2}
-            titleHelp={SurveyAnimalsI18N.animalFamilyHelp}
-            btnLabel={SurveyAnimalsI18N.animalFamilyAddBtn}
-            disableAddBtn={!lastAnimalValueValid('family', values)}
-            handleAddSection={() => push(newRelationship)}
-            handleRemoveSection={remove}>
-            {values.family.map((fam, index) => (
-              <Fragment key={fam._id}>
-                <Grid item xs={6}>
-                  <CbSelectWrapper
-                    label={'Family ID'}
-                    name={getAnimalFieldName<IAnimalRelationship>(name, 'family_id', index)}
-                    onChange={handleChange}
-                    controlProps={{
-                      size: 'small',
-                      required: isRequiredInSchema(AnimalRelationshipSchema, 'family_id')
-                    }}>
-                    {[
-                      ...(allFamilies ?? []),
-                      { family_id: newFamilyIdPlaceholder, family_label: newFamilyIdPlaceholder }
-                    ]?.map((a) => (
-                      <MenuItem disabled={!!disabledFamilyIds[a.family_id]} key={a.family_id} value={a.family_id}>
-                        {a.family_label ? a.family_label : a.family_id}
-                      </MenuItem>
-                    ))}
-                  </CbSelectWrapper>
-                </Grid>
-                <Grid item xs={6}>
-                  <Grid item xs={6}>
-                    <CbSelectWrapper
-                      label={'Relationship'}
-                      name={getAnimalFieldName<IAnimalRelationship>(name, 'relationship', index)}
-                      onChange={handleChange}
-                      controlProps={{
-                        size: 'small',
-                        required: isRequiredInSchema(AnimalRelationshipSchema, 'relationship')
-                      }}>
-                      <MenuItem key={'parent'} value={'parent'}>
-                        Parent in
-                      </MenuItem>
-                      <MenuItem key={'child'} value={'child'}>
-                        Child in
-                      </MenuItem>
-                    </CbSelectWrapper>
+    <Grid container spacing={3}>
+      <Grid item xs={12}>
+        <CbSelectWrapper
+          label={'Family ID'}
+          name={getAnimalFieldName<IAnimalRelationship>(name, 'family_id', index)}
+          onChange={handleChange}
+          controlProps={{
+            size: 'medium',
+            required: isRequiredInSchema(AnimalRelationshipSchema, 'family_id')
+          }}>
+          {[...(allFamilies ?? []), { family_id: newFamilyIdPlaceholder, family_label: newFamilyIdPlaceholder }]?.map(
+            (a) => (
```

Comment:
Fixed!

Reply:
+1. More specific is always better of course, but even in the worst case scenario something like `item` is still better imo, Just because you can at last search on `item` and reliably find its references, whereas `a` is really hard to identify reliably.